lib/views/linkedin_login_form.jade

```diff
@@ -15,7 +15,9 @@ script(type='text/javascript').
           serializedQueryString += '&';
         }
 
-        serializedQueryString += key + '=' + encodeURIComponent(value);
+        if (key !== 'access_token') {
```

Comment:
I'd have this for loop just continue, instead of wrapping `serializedQueryString += key + '=' + encodeURIComponent(value);`. Also add a comment to why it is neccessary to explicitly strip this query parameter from the URL.

E.g.

```javascript
// Because the access token isn't bla bla bla...
if (key === 'access_token') {
  continue;
}

serializedQueryString += key + '=' + encodeURIComponent(value);
```